code-challenges.test.js

```diff
@@ -1,5 +1,8 @@
 // ASSESSMENT 2: Coding Practical Questions with Jest
 
+//const { test, it } = require("@jest/globals")
+//const { describe } = require("yargs")
+
```

Comment:
These imports show up sometimes. You can always remove them if they cause issues.